Dockerfile.e2e

```diff
@@ -15,26 +15,29 @@ WORKDIR /go/src/github.com/docker/docker/
 
 # Generate frozen images
 COPY contrib/download-frozen-image-v2.sh contrib/download-frozen-image-v2.sh
-RUN contrib/download-frozen-image-v2.sh /output/docker-frozen-images \
+RUN ./contrib/download-frozen-image-v2.sh /output/docker-frozen-images \
```

Comment:
Was this related to the fix? If so, curious what changed, or if there's a regression

Reply:
well, it's just a "personal" pref 😓

Reply:
oh, it's fine to change, was just wondering if there was an issue there 😅